app/src/main/java/com/example/mylittlelibrary/AppModule.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.mylittlelibrary
+
+import com.example.mylittlelibrary.service.BookService
+import com.example.mylittlelibrary.service.IBookService
+import org.koin.androidx.viewmodel.dsl.viewModel
```

Comment:
Create AppModule.kt for the service and inteface. URL:https://docs.sonarqube.org/display/SONARQUBE45/Developers%27+Seven+Deadly+Sins